.github/workflows/nuget-publish.yml

```diff
@@ -49,6 +63,18 @@ jobs:
       - name: Check out ${{ github.repository }}
         uses: actions/checkout@v4
 
+      - name: Set package version
+        shell: pwsh
+        run: |
+          $PackageVersion = '${{ needs.preflight.outputs.package-version }}'
+          @("dotnet\Devolutions.NowClient\Devolutions.NowClient.csproj",
+            "dotnet\Devolutions.NowProto\Devolutions.NowProto.csproj") | ForEach-Object {
+            $csprojPath = $_
+            $csprojContent = Get-Content $csprojPath -Raw
+            $csprojContent = $csprojContent -Replace '(<Version>).*?(</Version>)', "<Version>$PackageVersion</Version>"
+            Set-Content -Path $csprojPath -Value $csprojContent -Encoding UTF8
+          }
```

Comment:
issue: We do not publish NowProto alone. Only NowAgent is relevant. NowProto is inlined into NowAgent for simplicity.